tests/cypress/e2e/actions_tasks/task_changes_status_after_initial_save.js

```diff
@@ -36,30 +37,29 @@ context('Delete a label from a task.', () => {
         cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
         cy.createZipArchive(directoryToArchive, archivePath);
         cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName);
-        cy.openTask(taskName);
     });
 
     after(() => {
-        cy.goToTaskList();
         cy.deleteTask(taskName);
     });
 
+    afterEach(() => {
+        cy.goToTaskList();
+    });
+
     describe(`Testing "${labelName}"`, () => {
         it('State of the created task should be "new".', () => {
+            cy.intercept('GET', /\/api\/users.*/).as('searchUsers');
```

Comment:
Why do we wait for this request here? In context of the test it would be more logical to wait for jobs request to complete or job cards to be visible. There is no connection with users